mods_cocina_mappings/mods_to_cocina_physicalDescription.txt

```diff
@@ -0,0 +1,111 @@
+1. Single physical description with all subelements
+<physicalDescription>
+  <form>ink on paper</form>
+  <reformattingQuality>access</reformattingQuality>
+  <internetMediaType>image/jpeg</internetMediaType>
+  <extent>1 sheet</extent>
+  <digitalOrigin>reformatted digital</digitalOrigin>
+  <note displayLabel="Condition">Small tear at top right corner.</note>
+</physicalDescription>
+{
+  "form": [
+    {
+      "value": "ink on paper",
+      "type": "form"
+    },
+    {
+      "value": "access",
+      "type": "reformatting quality",
+      "source": {
+        "value": "MODS reformatting quality terms"
+      }
+    },
+    {
+      "value": "image/jpeg",
+      "type": "media type",
+      "source": {
+        "value": "IANA media type terms"
+      }
+    },
+    {
+      "value": "1 sheet",
+      "type": "extent"
+    },
+    {
+      "value": "reformatted digital",
+      "type": "digital origin",
+      "source": {
+        "value": "MODS digital origin terms"
+      }
+    },
+    {
+      "note": [
+        {
+          "value": "Small tear at top right.",
+          "displayLabel": "Condition"
+        }
+      ]
+    }
+  ]
+}
+
+2. Multiple physical descriptions
+<physicalDescription>
+  <form>audio recording</form>
+  <extent>1 audiocassette</form>
+</physicalDescription>
+<physicalDescription>
+  <form>transcript</form>
+  <extent>5 pages</extent>
+</physicalDescription>
+{
+  "form": [
+    {
+      "value": "audio recording",
+      "type": "form"
+    },
+    {
+      "value": "1 audiocassette",
+      "type": "extent"
+    },
+    {
+      "value": "transcript",
+      "type": "form"
+    },
+    {

```

Comment:
This doesn't do a great job of showing that the extent of 5 pages belongs to the transcript and the 1 audiocasette belongs to the audio recording.  It would be hard to map back to the correct MODS from this.

Reply:
That's partly because this example reflects historic local practice for describing oral histories, but is suboptimal MODS (according to the DLF/Aquifer interoperability guidelines, physicalDescription is not a repeatable element). But I agree that making that distinction should be possible in the COCINA model, and I'll update the example accordingly.